asmcli/cloudbuild-main.yaml

```diff
@@ -174,6 +174,24 @@ steps:
       - '_CI_ASM_PKG_LOCATION=${_PKG_LOCATION}'
       - '_CI_ASM_KPT_BRANCH=${_ASM_PKG_BRANCH}'
     timeout: 600s
+  
+  - name: 'gcr.io/$PROJECT_ID/${_IMAGE_NAME}'
+    dir: 'asmcli'
+    id: 'run-add-to-mesh'
```

Comment:
I'm not sure if this is something we actually want to run on every pre-submit. Let's add it to the staging version now that it works.

Reply:
Done